src/file-events/cpp/win_fsnotifier.cpp

```diff
@@ -77,13 +86,6 @@ void WatchPoint::listen() {
 void WatchPoint::handleEvent(DWORD errorCode, DWORD bytesTransferred) {
```

Comment:
Let's remove the `errorCode` argument.

Reply:
Done as part of https://github.com/gradle/native-platform/pull/119.